### DISCUSSION TITLE
I'm having a problem with torch updating and I have no idea what I'm doing

### DISCUSSION THREAD

**Discussion by TheMisterMan12** [lllyasviel/Fooocus#3294] [Q&A] [answered]
I am a total noob when it comes to the programming side of Fooocus and am having a little bit of a problem with an updating or something. [Here](https://pastebin.com/Pfvi1HfB) is the log. Any help would be appreciated.

**Comment by mashb1t** [comment, marked as answer]
Did you follow https://github.com/lllyasviel/Fooocus/discussions/3293 "How to update"?

**Reply by TheMisterMan12** [reply]
Actually, didn't even see that had been posted yet, kinda just went straight to asking for help. I'll have a look. How do I open a terminal in a folder? I am a noob for Python stuff as well.

**Reply by mashb1t** [reply]
@TheMisterMan12 you can open a terminal in the current path by clicking in the address bar of the file explorer, typing `cmd` and hitting enter

**Reply by TheMisterMan12** [reply]
Thanks man, hopefully this can be used to help someone down the line.

**Comment by Godbaby-X** [comment]
error: subprocess-exited-with-error

  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> [6 lines of output]
      Traceback (most recent call last):
        File "<string>", line 2, in <module>
        File "<pip-setuptools-caller>", line 34, in <module>
        File "C:\Users\X\AppData\Local\Temp\pip-install-ixpxcdsv\groundingdino-py_87c478160ac0478eae90e3fd9281a41c\setup.py", line 41, in <module>
          readme = readme_file.read()
      UnicodeDecodeError: 'gbk' codec can't decode byte 0xa4 in position 2878: illegal multibyte sequence
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.

[notice] A new release of pip is available: 23.2.1 -> 24.1.2
[notice] To update, run: D:\Art-software\stable diffusion\Fooocus-new\python_embeded\python.exe -m pip install --upgrade pip

**Reply by mashb1t** [reply]
@Godbaby-X https://github.com/lllyasviel/Fooocus/discussions/3282#discussioncomment-10046423
